docs/faq.rst

```diff
@@ -42,45 +42,47 @@ Won't making a model fairer reduce its accuracy?
     To determine whether the new model is fairer, the whole system of which it is a part
     needs to be considered in terms of the societal context in which it is used.
-    See, for example, "Fairness and Abstraction in Sociotechnical Systems" by Selbst et al.,
-    listed on our :ref:`further_reading` page. 
-
-What sort of fairness related harms can Fairlearn address?
-    Fairlean concentrates on group fairness.
-    That is, obtaining the minimum disparity on some metric evaluated on
-    specified subgroups in the data.
+    In Fairlearn, we aim to combine a Python library of disparity metrics and mitigation algorithms
+    with a deep discussion of fairness in machine learning.
+    By considering this extra information in the context of your particular machine learning system,
+    you should be able to determine the most appropriate actions for your situation.
+
+What sort of fairness related harms can the Fairlearn library address?
+    Fairlearn concentrates on group fairness, meaning obtaining the minimum disparity on some
+    metric evaluated on specified subgroups in the data.
     This is certainly not the only way of looking at fairness.
     For example, there is the concept of individual fairness (where decisions are evaluated
     on the level of individuals rather than groups) and counter-factual fairness (e.g. does the
     decision change if an individual's gender is changed from male to female).
-    See :ref:`types_of_harms` for more information.
+    See :ref:`types_of_harms` for more information; we would welcome contributions which
+    enabled Fairlearn to help address some of these other harms.
     Beyond these, there are also fairness concepts which are note amenable to
     mathematical (and hence algorithmic) expression - justice, due process and
     righting historic iniquities for example.
 
-Can Fairlearn be used to detect bias in datasets?
+Can the Fairlearn library be used to detect bias in datasets?
     We do not have concrete plans for this at the present time.
 
-Can Fairlearn recommend ways to make my model fairer?
-    No. Fairlearn is not magic.
-
+Can the Fairlearn library recommend ways to make my model fairer?
     Fairness is a social concept, and no technological solution can make
     things fair automatically.
-    In order to use Fairlearn successfully, you first need to work out
+    In order to use the Fairlearn library successfully, you first need to work out
     what fairness means for your particular problem.
-    Once that is determined, you can then work out how to use the
-    various algorithms and fairness metrics which Fairlearn supports
-    to develop models which are *fairer by the standard you have chosen*.
+    We aim to provide a rich discussion of the topic of fairness in machine learning
+    on this website, together with links to the literature.
+    Once you have determined what fairness means for your machine pipeline,
```

Comment:
machine pipeline? Did you mean "machine learning pipeline"?

Usually the term AI system is preferable.

Reply:
Switched to AI system